cli/project-template/.botfront/botfront.yml

```diff
@@ -2,7 +2,7 @@ images:
   default:
     botfront: botfront/botfront:v0.21.3
     botfront-api: botfront/botfront-api:v0.21.3
-    rasa: botfront/rasa-for-botfront:v1.7.1-bf.3
+    rasa: botfront/rasa-for-botfront:v1.9.0-bf.1
     duckling: botfront/duckling:latest
     mongo: mongo:latest
     actions: rasa/rasa-sdk:1.7.0
```

Comment:
I think actions sdk must be bumped as well

Reply:
Done.